blockchain/state_manager/src/lib.rs

```diff
@@ -83,4 +83,17 @@ where
             ))
         }
     }
+    /// Returns a bls public key from provided address
+    pub fn get_bls_public_key(&self, addr: &Address, state_cid: &Cid) -> Result<Vec<u8>, Error> {
```

Comment:
Just a note that the public key should probably be an array (I'll make a pass through the Address crate soon I guess)